crates/biome_js_analyze/src/lint/a11y/no_label_without_control.rs

```diff
@@ -159,14 +152,124 @@ impl Rule for NoLabelWithoutControl {
 
 #[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
```

Comment:
Using the `serde(default)` attribute should be enough to make optional he fields.
Is there an interest to have both `Option<>` and `serde(default)`?

Reply:
Agreed it might make more sense to use plain `Vec`s only. It uses less memory, and makes consuming code easier. The only exception should be when an empty vector and `None` are semantically different.

Reply:
If i am not mistaken, schemars recognizes `serde(default)`

Reply:
Done, I updated the code and the contribution guide to use `serde(default)`